don't try to create a venv with conda, because we don't have it

diff --git a/tests/test_sessions.py b/tests/test_sessions.py
index 98478a6..97fb4a2 100644
--- a/tests/test_sessions.py
+++ b/tests/test_sessions.py
@@ -948,3 +948,2 @@ class TestSessionRunner:
             ("nox.virtualenv.VirtualEnv.create", "venv", nox.virtualenv.VirtualEnv),
-            ("nox.virtualenv.CondaEnv.create", "conda", nox.virtualenv.CondaEnv),
         ],
